src/sentry/api/serializers/rest_framework/dashboard.py

```diff
@@ -59,6 +60,11 @@ class QueryWarning(TypedDict):
     columns: dict[str, str]
 
 
+class LinkedDashboard(TypedDict):
+    field: str
+    dashboard_id: int
+
+
```

Comment:
Just curious, but why do you need this `TypedDict` _and_ the serializer?

Reply:
I don't know if it's common to pass serialized objects around, but it seems like it'd save you some work, like checking whether the `LinkedDashboard` objects all have a field and a dashboard ID lower in the code